config/routechangerconfig.go

```diff
@@ -6,17 +6,27 @@ import (
 	"strings"
 )
 
-type routeChangerConfig struct {
+type routeChangerConfigItem struct {
 	Reg  *regexp.Regexp
 	Repl string
 	Sep  string
 }
 
+type routeChangerConfig []routeChangerConfigItem
+
 func (rcc routeChangerConfig) String() string {
-	if rcc.Reg == nil {
-		return ""
+	result := ""
+	for _, rcci := range rcc {
+		if rcci.Reg != nil {
```

Comment:
Why Reg could be nil?

Reply:
I think it could not. However, this check is present in code before my changes so I saved it. Okay will remove.